sqjobs/tests/tests.py

```diff
@@ -0,0 +1,172 @@
+from django.test import TestCase
+from ..brokers.broker import Standard
+from ..brokers.eager import Eager
+from sqjobs.contrib.django.djsqjobs.beat import Beat
+from sqjobs.contrib.django.djsqjobs.models import PeriodicJob
+from sqjobs.contrib.django.djsqjobs.models import JobStatus
+
+from .fixtures import Adder, Divider
+from .django_fixtures import ResultDivider
+from ..connectors.dummy import Dummy
+
+from datetime import datetime
+import pytz
+
+from mock.mock import patch
+
+
+class DjangoTestCase(TestCase):
+    fixtures = ['sqjobs/tests/djsqjobs_tests-fixture.json']
+
+    def setUp(self):
+        from django.core.management.commands.loaddata import Command
+
+        loaddata_command = Command()
+        loaddata_command.using = 'default'
+        loaddata_command.ignore = None
+        loaddata_command.app_label = None
+        loaddata_command.verbosity = 0
+        loaddata_command.hide_empty = None
+        loaddata_command.loaddata(self.fixtures)
+
+
+class PeriodicJobsTests(DjangoTestCase):
+    beat = Beat(None, 10, True)
+    beat.register_job(Adder)
+
+    def test_get_expired_jobs(self):
+        assert len(self.beat.get_expired_jobs()) == 2
+
+    def test_add_delayed_job_info_and_programmed_date(self):
+        expired_job = PeriodicJob.objects.get(pk=2)
+        expired_job_kwargs = self.beat.get_job_kwargs(expired_job)
+        kwargs_with_delayed_info = self.beat.add_delayed_job_info(
+            expired_job, expired_job_kwargs
+        )
+        assert kwargs_with_delayed_info['num1'] == 1
+        assert kwargs_with_delayed_info['num2'] == 2
+        assert kwargs_with_delayed_info[PeriodicJob.PROGRAMMED_DATE] == expired_job.next_execution.astimezone(pytz.timezone(expired_job.timezone)).isoformat()
+        assert kwargs_with_delayed_info[PeriodicJob.DELAYED_JOB]
```

Comment:
Use `self.assertEqual(kwargs_with_delayed_info['num1'], 1)` and others, they produce friendlier error messages.